src/utils/dates.js

```diff
@@ -10,3 +10,34 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+/**
+ * Calculates the number of days between two JS Date objects.
+ * @example
+ * // Returns the number of days that have passed between the two dates.
+ * const startDate = new Date("2023-09-15");
+ * const endDate = new Date("2023-09-24");
+ * getDaysBetweenDates(startDate, endDate); // Output is 9 (days)
+ * @param {Date} startDate - The starting date.
+ * @param {Date} endDate - The ending date.
+ * @returns {number} - The number of days between the start and end dates.
+ */
+export function getDaysBetweenDates(startDate, endDate) {
+	try {
+		if (startDate instanceof Date && endDate instanceof Date) {
+			const days_between_in_milliseconds =
```

Comment:
Minor nitpick: This variable should be in camel case as per convention.
`ONE_DAY_IN_MILLISECONDS` is named as a [constant](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/) according to that convention

Reply:
@mentalcaries Ok, I changed `days_between_in_milliseconds` to camel case, and now it's `daysBetweenInMilliseconds`. We initially used snake case because it was slightly unreadable. Thank you for pointing that out!